cnxarchive/tests/testing.ini

```diff
@@ -4,7 +4,7 @@ db-connection-string = dbname=cnxarchive-testing user=cnxarchive password=cnxarc
 memcache-servers = localhost
 search-cache-expiration = 15
 search-long-cache-expiration = 30
-
+sitemap-destination = http://cnx.org/sitemap.xml
```

Comment:
@karenc it's possible but I don't see the benefit of doing that. We want to point the sitemap to another url's sitemap. If we auto generate it by using the root url like: archive.cnx.org ==> cnx.org, then it will break if archive.cnx ever move to another domain.